ci/images/alpine-tools/Dockerfile

```diff
@@ -22,11 +22,20 @@ RUN apk --no-cache add \
       bash \
       curl \
       git \
+      go \
       jq \
+      make \
+      musl-dev \
       openssh-client \
+      openssl \
       python \
       py2-pip \
       rsync \
   && gcloud config set core/disable_usage_reporting true \
   && gcloud config set component_manager/disable_update_check true \
-  && gcloud config set metrics/environment github_docker_image
\ No newline at end of file
+  && gcloud config set metrics/environment github_docker_image \
```

Comment:
I'd be more inclined to have a alpine-tools-windows image so that the commonly used image doesn't have the all this stuff just to support a single task.

Reply:
I agree, you can make the image FROM alpine-tools then add the extra stuff, then create an image job that depends on alpine-tools.

Reply:
Seems like so much unnecessary work. Why not just keep it simple?

Reply:
What is the total size of the image compared to the original image. We have seen task start times relative to the size of the image.